testcrate/src/lib.rs

```diff
@@ -6,10 +6,15 @@ extern "C" {
     pub fn OpenSSL_version_num() -> c_ulong;
 }
 
-#[test]
-fn version_works() {
-    unsafe {
-        println!("{:#x}", OpenSSL_version_num());
-        assert!(OpenSSL_version_num() > 0);
+#[cfg(test)]
+mod tests {
+    use super::OpenSSL_version_num;
+
+    #[test]
+    fn version_works() {
+        unsafe {
+            println!("{:#x}", OpenSSL_version_num());
+            assert!(OpenSSL_version_num() > 0);
+        }
     }
```

Comment:
I think this file change may not be necesary?

Reply:
I can revert it if needed -- I just thought it was weird to have a test not wrapped in the usual conditional compilation directive/mod combo. Kinda instinctive fix 🤔

Reply:
Reverted!